lib/public/views/Runs/RunPerDataPass/RunsPerDataPassOverviewPage.js

```diff
@@ -65,20 +69,26 @@ export const RunsPerDataPassOverviewPage = ({ runs: { perDataPassOverviewModel }
                 : {},
             Other: () => {},
         }),
-        ...createRunDetectorsAsyncQcActiveColumns(remoteDetectors.match({
-            Success: (payload) => payload,
-            Other: () => [],
-        }), {
-            profiles: 'runsPerDataPass',
-            dataPassId: remoteDataPass.match({
-                Success: (dataPass) => dataPass.id,
-                Other: () => null,
+        ...createRunDetectorsAsyncQcActiveColumns(
+            remoteDetectors.match({
```

Comment:
Once again, I did not noticed the other in view.
This is once again bad, because if something goes wrong or is loading, the user is not informed about it.
If any of the remote data is failure, the page should probably display an error message. If it's loading, there should be a spinner somewhere (maybe a full page spinner, maybe only in the table).
This applies to all the remote data in this page.
Once again, you should almost NEVER use Other in views!